kubernetes-client/src/main/java/io/fabric8/kubernetes/client/utils/Utils.java

```diff
@@ -471,4 +473,11 @@ public static String getSystemPathVariable() {
   private static String getOperatingSystemFromSystemProperty() {
     return System.getProperty(OS_NAME);
   }
+
+  public static boolean isHttpStatusCodeFromErrorEncounteredByServer(int code) {
```

Comment:
Since this is not checking if the HTTP status code belongs to a server error family (`code >= 500`), and for our use-case we just want to retry if one of these status codes is returned by the server, it's probably a good idea to move the method to `CreateOrReplaceHelper` class.
Maybe also rename it to `shouldRetry` or something like that, which suggests that the procedure is only retried for some specific status codes.